CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ project(QuEST_Project)
 # ----- USER OPTIONS ----------------------------------------------------------
 # -----------------------------------------------------------------------------
 
-set(USER_SOURCE  "tutorial_example.c"  CACHE STRING "Source to build with QuEST library")
+set(USER_SOURCE  "examples/tutorial_example.c"  CACHE STRING "Source to build with QuEST library")
```

Comment:
Doing this invalidates the current README, which includes a copy:
```bash
cp examples/tutorial_example .
mkdir build
cd build
cmake ..
make
```
However, I prefer your method here. Can you remove the `cp examples/tutorial_example .` line from the README?

Reply:
Removed line in acf7f95214544204939c8267381218a72ec98096.